package.json

```diff
@@ -74,10 +73,6 @@
     "metro-source-map": "^0.66.2",
     "metro-symbolicate": "^0.66.2",
     "metro-transform-plugins": "^0.66.2",
-    "metro-transform-worker": "^0.66.2",
-    "react": "17.0.1",
-    "react-native": "^0.64.3",
-    "react-native-windows": "^0.64.3",
-    "react-native-macos": "^0.64.6"
```

Comment:
Can we actually keep these (and react-native-win32 above) in? I like it better this way 😅